DigitalLearningSolutions.Data/DataServices/LearningResourceReferenceDataService.cs

```diff
@@ -27,5 +28,15 @@ FROM LearningResourceReferences
                 new { learningResourceReferenceId }
             ).Single();
         }
+
+        public string? GetLearningHubResourceLinkById(int learningResourceReferenceId)
+        {
+            return connection.Query<string?>(
+                @"SELECT ResourceLink
+                    FROM LearningResourceReferences
+                    WHERE ID = @learningResourceReferenceId",

```

Comment:
Should this be searching on `ID` or `ResourceRefID` ?

Reply:
Now you mention it, I think ResourceRefID, and I'll implement that, but I'm not totally sure.

Reply:
Looks like it should definitely be ResourceRefID from the update on the ticket.